project/make/.ensure-webserver

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+# Build a Go static web server on top of busybox image
+# and compile it for target daemon
+IMAGE="webserver"
+
+dir="$DEST/webserver"
+mkdir -p "$dir"
+(
+	cd "$dir"
+	cat > main.go <<'EOF'
+		package main
+
+		import (
+		  "log"
+		  "net/http"
+		)
+
+		func main() {
+		  fs := http.FileServer(http.Dir("/static"))
+		  http.Handle("/", fs)
+		  log.Panic(http.ListenAndServe(":80", nil))
+		}
```

Comment:
Did you forget to push or something?

Replacing `cat` with `gofmt` isn't at all what I meant, either.  I meant that the code in this file itself should be `gofmt`ted appropriately, even though we can't verify the embedded generated code.